src/style/demo-styling.css

```diff
@@ -56,68 +58,77 @@ html {
 }
 
 .bttn.primary{
+  display: inline-block;
+  width: max-content;
```

Comment:
I needed to add this due to the box shadow not working properly in Safari specifically. It appears that for some reason that the `inset` property only worked for block level elements. And to make sure that the button stays in one line I made the width only expand till `max-content`

Reply:
Thanks for catching that 💜